### PR TITLE
Cockpit 203

### PR DESCRIPTION
**Pull request by mvollmer**
https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-203.html

### CODE REVIEW

**Comment by mvollmer** [comment]
The Bodhi links need to be added.

**Comment by mvollmer** [comment]
Bodhi links have been added.

File: _posts/2019-09-18-cockpit-203.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+category: release
+summary: Cockpit with Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 203.
+
+### Machines: Network Interfaces
+
+It is now possible to add network interfaces to virtual machines.
```
Comment (garrett) [inline _posts/2019-09-18-cockpit-203.md]:
```suggestion
Virtual machines now support network interface selection.
```

**Comment by garrett** [comment]
I've also optimized the PNG (please squash the commit).

File: _posts/2019-09-18-cockpit-203.md (hunk 1 above)
Comment (mvollmer) [reply _posts/2019-09-18-cockpit-203.md]: @garrett, afaik, this is not about selecting interfaces but actually creating new ones.  Like buying a new network card, but it's all virtual.

Comment (mvollmer) [reply _posts/2019-09-18-cockpit-203.md]: I have changed it to "Virtual machines now support creation of virtual network interfaces."

**Pull request merged**